CMakeLists.txt

```diff
@@ -2,16 +2,19 @@
 cmake_minimum_required(VERSION 3.18)
 
 #set up a variable to hold the project name
-set(PROJ_NAME "ReportRepair")
+set(PROJ_NAME "ReportRepair"
+    CACHE STRING "sets up the project name")
 
 #declare a new project
 project(${PROJ_NAME})
 
 #tell Cmake that when looking for a compiler, use one that conforms to CPP14
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 14
+    CACHE STRING "flags for CPP 14 standard")
 
-#make sure all the things use headers in src/
-include_directories(src)
+#enable unit testing
```

Comment:
apparently this is really important. Have to make sure that `include(CTest)`, which automatically calls `enable_testing()`, before adding any subdirectories.